core/controllers/base.py

```diff
@@ -438,7 +438,7 @@ def validate_and_normalize_args(self):
                 handler_args, schema_for_request_method, extra_args_are_allowed,
                 allow_string_to_bool_conversion)
         )
-
+  
```

Comment:
Please remove the whitespace characters here and below.

Reply:
done